src/processing.integration-test.ts

```diff
@@ -152,36 +140,141 @@ describe("processProposals", () => {
       happenedAt: new Date("2024-03-20T09:48:23.000Z"),
     };
 
-    const spaceWithRealOracle = {
-      ...space,
-      oracleAddress: ONEINCH_ORACLE_ADDRESS,
-    };
+    let space: Space;
+
+    beforeEach(async () => {
+      space = {
+        ...randomizeSpace(),
+        oracleAddress: ONEINCH_ORACLE_ADDRESS,
+      };
+      await insertSpaces([
+        {
+          ens: space.ens,
+          startBlock: 1n,
+          lastProcessedBlock: 1n,
+        },
+      ]);
+    });
 
-    await fn({
-      space: spaceWithRealOracle,
-      proposals: [proposal],
-      getLogNewQuestionFn: getLogNewQuestion,
-      notifyFn,
-      insertProposalFn: insertProposal,
+    afterEach(async () => {
+      await removeProposalByQuestionId(proposal.questionId);
+      await removeSpaceByEns(space.ens);
```

Comment:
Promise.all can't be use here due to the database relationship between proposal and space